src/lib/_imports/trumps/s-header.css

```diff
@@ -146,17 +161,13 @@ Styleguide Trumps.Scopes.Header
 /* Navigation: Links: Responsive Design */
 
 /* Tweak Bootstrap `_nav.scss` (which selects via `navbar-expand-` class) */
-.s-header[class*="navbar-expand-"] .navbar-nav .nav-link {
-  padding-top: 8px; /* NO-R/EM: 0.5rem (from Bootstrap) */
-  padding-bottom: 8px; /* NO-R/EM: 0.5rem (from Bootstrap) */
-
-  padding-right: 14px; /* NO-R/EM: .875rem (overwrite Bootstrap) */
-  padding-left: 14px; /* NO-R/EM: .875rem (overwrite Bootstrap) */
-}
-
-@media only screen and (min-width: 992px) {
+@media (--medium-and-below) {
   .s-header[class*="navbar-expand-"] .navbar-nav .nav-link {
-      padding-block: 0; /* overwrite core-styles.header.css */
+    padding-top: 8px; /* NO-R/EM: 0.5rem (from Bootstrap) */
+    padding-bottom: 8px; /* NO-R/EM: 0.5rem (from Bootstrap) */
+  
+    padding-right: 14px; /* NO-R/EM: .875rem (overwrite Bootstrap) */
+    padding-left: 14px; /* NO-R/EM: .875rem (overwrite Bootstrap) */
   }
 }
```

Comment:
**For _vertical_ padding, perfect.**

**But** I think _horizontal_ padding is supposed to alway be active, not just at `--medium-and-below`... your screenshots seem to show it active on wide screen... My "spidey sense" is tingling. I'll ignore until next week.

Reply:
Whoops. Ignore. _I was reviewing old code._